src/test/cli/ceph-authtool/cap-bin.t

```diff
@@ -2,5 +2,5 @@
   creating kring
 
   $ ceph-authtool --cap osd 'allow rx pool=swimming' kring
-  $ ceph-authtool kring --list|grep -P '^\tcaps '
+  $ ceph-authtool kring --list|grep -E '^\Wcaps '
```

Comment:
@tchaikov 
-P is perl regex, and that is not in FreeBSD grep.
And `\t`is perl-speak.
So I try to stick to POSIX

Perhaps a bit of a braintwist at my end, but `\W` is abit more liberal in that it catches all types of white space. So it for whatever reason the output changes a bit, this will match that too.